tests/functional/heketi/test_heketi_brick_evict.py

```diff
@@ -111,19 +113,25 @@ def _wait_for_gluster_pod_after_node_reboot(self, node_hostname):
                 self.ocp_client, gluster_pod, service, "active", state)
 
     @pytest.mark.tier4
-    def test_brick_evict_with_node_down(self):
+    @ddt.data("three_nodes", "more_than_three_nodes")
+    def test_brick_evict_with_node_down(self, node_count):
         """Test brick evict basic functionality and verify brick evict
         after node down"""
 
         h_node, h_server = self.heketi_client_node, self.heketi_server_url
 
-        # Disable node if more than 3
+        # Get available nodes from cluster
         node_list = heketi_ops.heketi_node_list(h_node, h_server)
-        if len(node_list) > 3:
-            for node_id in node_list[3:]:
-                heketi_ops.heketi_node_disable(h_node, h_server, node_id)
-                self.addCleanup(
-                    heketi_ops.heketi_node_enable, h_node, h_server, node_id)
+
+        if node_count == "three_nodes":
```

Comment:
I'm bit confused here. If three nodes then disabling other nodes?
what about more than three nodes in other ddt?

Reply:
There are two scenarios in TC itself. If there are three nodes, the brick evict creation will fail and if more than three nodes then the brick evict will get success.  We don't want to check just three nodes or more than three nodes and skipping test cases. We thought to disable the remaining node if there are three nodes available.

Reply:
Yes, if there are more than 3 nodes that were being handled by the previous code.
And if you are passing 3 nodes as `three_nodes`  then I don't get why to disable as we have already 3 nodes.

Reply:
@adityaramteke Yes, if we have 4 nodes, so there we disable all the remaining node apart from the 3 nodes